tests/integration/transitions/sentinel-api-transitions.spec.js

```diff
@@ -1055,4 +1055,73 @@ describe('transitions', () => {
         chai.expect(updatedPerson.patient_id).to.be.ok;
       });
   });
+
+  it('calling getDeprecatedTransitions endpoint does not break API transitions', () => {
```

Comment:
Someone in the future is going to laugh when they read this test and think "I wonder what horrible bug lead to this test case...."  😂

Reply:
You're right. I left comments with the issue number in the past, but I found I just `git blamed` to get info about the code and seldom copied the issue number (or link) and pasted it in the browser. 

What do you prefer? 
- a comment mentioning the issue number 
- renaming the test (the bloody endpoint needs at least one e2e test!)
- no change

Reply:
IMHO git blame is sufficient for doing historical research into a code change.  (That is kinda the whole point of commit history...)  

Sorry, I should have been more clear that I do not think anything actually needs to change here.  I was just amused at how the most nonsensical test cases always result from weird bugs (and anytime you see a test case that seems really random, it is probably the result of an issue like this...).